### PR TITLE
Django Girls update

### PR DESCRIPTION
**Pull request by ntoll**
This branch makes two changes:
- I've added some clarity to the registration form.
- I've included links to the assessment criteria Django Girls uses.

### CODE REVIEW

File: content/djangogirls.html [hunk 1]
```diff
@@ -18,5 +18,7 @@
 with empathy.</p>
 
 <p>To register your interest in attending please
-<a href="http://djangogirls.org/coventry/apply/">fill in this form</a>.</p>
+<a href="http://djangogirls.org/coventry/apply/">fill in this form</a>.
+<a href="http://blog.djangogirls.org/post/91067112853/djangogirls-how-we-scored-applications">This blog post</a>
```
Comment (doismellburning) [inline content/djangogirls.html]: Can you please use better link text for a11y? https://twitter.com/NeilMilliken/status/610568082382262274

File: content/register.html [hunk 2]
```diff
@@ -12,8 +12,15 @@
 
 <p>To register your interest in attending as a sponsored
 <a href="http://trans-code.org/">Trans*Code</a> participant
-please <a href="https://docs.google.com/forms/d/1uoGFiJAlwg4r0WycpwdvuveaLq_fIdfU3Dq0SdBgdCQ/viewform">use this form</a> and we'll be in touch.</br >
-To register as a <a href="http://djangogirls.org/coventry/">Django Girls</a> participant please <a href="http://djangogirls.org/coventry/apply/">fill in this form</a>.</p>
+please <a href="https://docs.google.com/forms/d/1uoGFiJAlwg4r0WycpwdvuveaLq_fIdfU3Dq0SdBgdCQ/viewform">use this form</a> and we'll be in touch.</p>
+<p>To register as a <a href="http://djangogirls.org/coventry/">Django Girls</a>
+participant please <a href="http://djangogirls.org/coventry/apply/">fill in
+this form</a>.
+<a href="http://blog.djangogirls.org/post/91067112853/djangogirls-how-we-scored-applications">This blog post</a>
```
Comment (doismellburning) [inline content/register.html]: https://github.com/PyconUK/pyconuk.org/pull/42/files#r33759471

File: content/djangogirls.html (hunk 1 above)
Comment (ntoll) [inline content/djangogirls.html]: +1 fixing now.

**Comment by doismellburning** [comment]
Much better, cheers. What're your thoughts on moving whole sentence inside tag? (Not an objection, happy to merge as is, just curious)

**Comment by ntoll** [comment]
More a questions of aesthetics in terms of both the language and how the link looks within the sentence (I dislike looooooooong links). ;-)

**Comment by ntoll** [comment]
Please merge... (then I can announce Django Girls via Twitter etc....)

**Pull request merged**